site/templates/footer.jet

```diff
@@ -1,9 +1,9 @@
 {{block footer()}}
-	<footer>
+	<footer role="contentinfo">
```

Comment:
[Looks to be redundant](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Contentinfo_role).

```suggestion
	<footer>
```